app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -10,19 +10,19 @@ import android.view.*
 import android.widget.Toast
 import kotlinx.android.synthetic.main.fragment_search.view.*
 import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.event.EVENT_ID
```

Comment:
Not required I guess

Reply:
that was done automatic by Android studio, i have `optimize imports on fly active` this happened because line 21 got removed